src/staverepetition.ts

```diff
@@ -192,7 +192,9 @@ export class Repetition extends StaveModifier {
       this.y_shift +
       Tables.currentMusicFont().lookupMetric('staveRepetition.symbolText.offsetY');
     if (draw_coda) {
-      Glyph.renderGlyph(ctx, symbol_x, y, 40, 'coda', { category: 'coda' });
+      Glyph.renderGlyph(ctx, symbol_x, y, Font.convertSizeToPointValue(this.textFont?.size) * 2, 'coda', {
```

Comment:
Maybe the hardcoded 2 should be customizable in the future? Perhaps I don't want my coda to be that huge? I could set it to 1 instead....